### PR TITLE
feat (DOCS-CODE-027): [sign-in] WinUI 3 Packaged App canonical sample

### PR DESCRIPTION
**Pull request by ferantivero**

### CODE REVIEW

File: src/sign-in-winuiapp/MainWindow.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            tokenAcquisitionTextBlock.Text = $"Access Token: {(tokenWasFromCache ? "Cached" : "Newly Acquired")} (Expires: {msalAuthenticationResult.ExpiresOn:R})";
+
+            this.signInCallToActionTextBlock.Visibility = Visibility.Collapsed;
+            this.authorizedPanel.Visibility = Visibility.Visible;
+        }
+
+        private async void SignOut_Click(object sender, RoutedEventArgs e)
+        {
+            // Signing out is removing all cached tokens, meaning the next token request will
+            // require the user to sign in.
```
Comment (skabou) [inline src/sign-in-winuiapp/MainWindow.xaml.cs:97]:
```suggestion
            // All cached tokens will be removed.
            // The next token request will require the user to sign in.

```

File: src/sign-in-winuiapp/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Open the project in Visual Studio
+
+Next, open the _MsalExamples.csproj_ project in Visual Studio.
+
+### 3. Update code sample in _MainWindow.cs_ with app registration values
+
+Finally, set the following values in _MainWindow.xaml.cs_.
```
Comment (skabou) [inline src/sign-in-winuiapp/README.md:56]: Should these both be _MainWindow.cs_ or is it intentional for them to be different?

**Review by skabou — COMMENTED** [review]
I see this is using "PublicClientApplicationBuilder"
and RedirectURI of "https://login.microsoftonline.com/common/oauth2/nativeclient"

I'm not sure but on AAD have you tried removing RedirectURI and instead enabling "Allow public client flows" ?

File: src/sign-in-winuiapp/README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+languages:
+- csharp
+page_type: sample
+name: "WinUI 3 Packaged Desktop App that makes a request to the Graph API after signing in the user"
+description: "This .NET 6 (C#) WinUI 3 Packaged Desktop App signs in the user and then makes a request to Microsoft Graph for the user's profile data."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-app-csharp-winforms
```
Comment (ckittel) [inline src/sign-in-winuiapp/README.md:14]:
```suggestion
urlFragment: ms-identity-docs-code-app-csharp-winui
```

File: src/sign-in-winuiapp/README.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+This .NET 6 WinUI 3 Packaged Desktop App presents a button that initiates an authentication flow using the Microsoft Authentication Library (MSAL). The user completes this flow in their default web browser. Upon successful authentication, an HTTP GET request to the Microsoft Graph /me endpoint is issued with the user's access token in the HTTP header. The response from the GET request is then displayed to the user. The MSAL client first looks to its token cache, refreshing if necessary, before acquiring a new access token.
+
+## Reporting problems
+
+### Sample app not working?
+
+If you can't get the sample working, you've checked [Stack Overflow](http://stackoverflow.com/questions/tagged/msal), and you've already searched the issues in this sample's repository, open an issue report the problem.
+
+1. Search the [GitHub issues](/issues) in the repository - your problem might already have been reported or have an answer.
+1. Nothing similar? [Open an issue](/issues/new) that clearly explains the problem you're having running the sample app.
```
Comment (ckittel) [inline src/sign-in-winuiapp/README.md:85]:
```suggestion
1. Search the [GitHub issues](../../../issues) in the repository - your problem might already have been reported or have an answer.
1. Nothing similar? [Open an issue](../../../issues/new) that clearly explains the problem you're having running the sample app.
```

**Review by ckittel — APPROVED** [review]
Only one small change that should be made (clarity on the redirect URI), otherwise, everything else is just nits.

File: src/sign-in-winuiapp/README.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+Use these settings in your app registration.
+
+| App registration <br/> setting  | Value for this sample app                                           | Notes                                                                           |
+|--------------------------------:|:--------------------------------------------------------------------|:--------------------------------------------------------------------------------|
+| **Name**                        | `active-directory-dotnet-winui3`                                    | Suggested value for this sample. <br/> You can change the app name at any time. |
+| **Supported account types**     | **Accounts in this organizational directory only (Single tenant)**  | Suggested value for this sample.                                                |
+| **Platform type**               | **Mobile and desktop applications**                                 | Required value for this sample                                                  |
+| **Redirect URIs**               | `https://login.microsoftonline.com/common/oauth2/nativeclient`      | Required value for this sample                                                  |
```
[the remark this answers is not in the document]
Comment (ferantivero) [reply src/sign-in-winuiapp/README.md:46]: yup it is + good catch @ckittel, 

done | addressed from 16aeed1ef3df7ec986cd98dd0d6a3bf02a0d25c2

**Comment by skabou** [comment]
@ferantivero ok I guess that is not the way... Thanks for giving it a shot though!

**Comment by ckittel** [comment]
Skipping CD review per standup's guidance yesterday.  Merging!

**Pull request merged**